pkg/plugins/golang/v2/webhook.go

```diff
@@ -101,7 +101,7 @@ func (p *createWebhookSubcommand) InjectResource(res *resource.Resource) error {
 			return fmt.Errorf("%s create webhook requires a previously created API ", p.commandName)
 		}
 	} else {
-		if r, err := p.config.GetResource(p.resource.GVK); err != nil {
+		if r, err := p.config.GetResource(p.resource.GVK); err != nil || !r.HasAPI() {
```

Comment:
IHMO we need here to improve this check. 

If the error be faced here because the resource was not found we need to check if the resource is a core type and it is scaffold in the project file. If yes, then fine we should not return an error and allow scaffold the webhooks.